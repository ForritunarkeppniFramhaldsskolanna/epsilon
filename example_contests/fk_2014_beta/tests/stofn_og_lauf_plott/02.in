40
69 52 43 72 88 27 46 63 59 45 16 53 94 56 76 53 19 26 40 72 57 85 75 63 78 41 79 27 86 49 67 30 49 57 65 27 73 39 53 37
