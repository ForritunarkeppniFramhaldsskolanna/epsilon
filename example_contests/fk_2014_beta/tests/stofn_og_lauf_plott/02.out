1 69
2 6777
3 079
4 0135699
5 23336779
6 33579
7 2235689
8 568
9 4
